src/python/pants/base/exiter.py

```diff
@@ -107,7 +105,7 @@ def _log_exception(self, msg):
           exception_log.write('timestamp: {}\n'.format(datetime.datetime.now().isoformat()))
           exception_log.write('args: {}\n'.format(sys.argv))
           exception_log.write('pid: {}\n'.format(os.getpid()))
-          exception_log.write(msg.encode('utf-8'))
+          exception_log.write(msg)
```

Comment:
The file `output_path`, which == `work_dir/logs/exceptions.log`, is opened in text mode. 

I don't think it matters much if we choose to open it in binary vs text, it seems like a small contained change, and either works.